lib/puppet/functions/parsehocon.rb

```diff
@@ -4,15 +4,14 @@
 #   This function accepts HOCON as a string and converts it into the correct
 #   Puppet structure
 #
-# @return
-#   Data
-#
 # @example How to parse hocon
 #   $data = parsehocon("{any valid hocon: string}")
 #
 Puppet::Functions.create_function(:parsehocon) do
   # @param hocon_string A valid HOCON string
   # @param default An optional default to return if parsing hocon_string fails
+  # @return
+  #   Data
```

Comment:
I thought there was a return type directive but https://puppet.com/docs/puppet/7/functions_ruby_overview.html doesn't show it. Am I confused or is there a Ruby native way to specify the same thing?

Reply:
I moved `@return Data` but it should be `@return [Data]`… Fixing